swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Bulletin'
+
+  /bulletins/{id}/editions:
+    get:
+      tags:
+        - "Public"
+      summary: Get a list of editions of a bulletin
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/EditionsList'
+
+  /bulletins/{id}/editions/{edition}:
+    get:
+      tags:
+        - "Public"
+      summary: Get an edition of a bulletin
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/edition'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Edition'
+
+  /bulletins/{id}/editions/{edition}/versions:
+    get:
+      tags:
+        - "Public"
+      summary: Get a list of versions of an edition
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/edition'
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+      description: (future implementation)
+      produces:
+        - application/json
+      responses:
+        501:
+          description: Not Implemented
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/VersionsList'
+
+  /bulletins/{id}/editions/{edition}/versions/{version}:
+    get:
+      tags:
+        - "Public"
+      summary: Get a version
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/edition'
+        - $ref: '#/parameters/version'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Version'
+
+parameters:
+  id:
+    name: id
+    description: ID of a bulletin
+    type: string
+    in: path
+    required: true
+  edition:
+    name: id
+    description: Edition of a bulletin
+    type: string
+    in: path
+    required: true
+  version:
+    name: id
+    description: Version of a bulletin
+    type: string
+    in: path
+    required: true
+  limit:
+    name: limit
+    description: The number of items requested, defaulted to 50 and limited to 1000.
+    type: integer
+    required: false
+    in: query
+  offset:
+    name: offset
+    description: The first row of resources to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter.
+    type: integer
+    required: false
+    in: query
 
 definitions:
-  HelloResponse:
+  BulletinsList:
+    type: object
+    properties:
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Bulletin'
+
+  Bulletin:
+    type: object
+    properties:
```

Comment:
The bulletin and version should support welsh and english.  We can use the `lang` query param select the language, we can also return a `language` field on these endpoints for the language returned to make it clear when english is being returned because no welsh is available.